src/common/xo/index.js

```diff
@@ -634,6 +634,60 @@ export const editNetwork = ({ id }, props) => (
   call('network.set', { ...props, id })
 )
 
+import CreateNetworkModalBody from './create-network-modal'
+export const createNetwork = container => (
```

Comment:
What is this `container` parameter?

`xo` methods should not depend on the UI.